src/js/disability-benefits/components/AddFilesForm.jsx

```diff
@@ -105,12 +105,18 @@ class AddFilesForm extends React.Component {
           <div key={index} className="document-item-container">
             <Element name={`documentScroll${index}`}/>
             <div className="document-title-size">
-              <div className="document-title-header">
-                <h4 className="title">{file.name}</h4>
+              <div className="document-title-text-container">
+                <div className="document-title-header">
+                  <h4 className="title">{file.name}</h4>
```

Comment:
Good call, I added a width and a word-break property so the text won't go outside of the div.
![screen shot 2016-10-26 at 1 35 02 pm](https://cloud.githubusercontent.com/assets/6254227/19737451/81807c40-9b81-11e6-9524-28d9ca0d07f8.png)